docs/CodeStyle.md

```diff
@@ -17,5 +17,97 @@ limitations under the License.
 
 # NIXL Code style
 
-* Lower camel style: Data types, Classes, Class Members, Member functions
-* Snake style: function arguments, local variables
\ No newline at end of file
+## Naming Conventions
+
+* **Lower camel case** (e.g., `myVariable`): Data types, Classes, Class Members, Member functions
```

Comment:
@w1ldptr i wonder if you know a tool that can enforce it ?

Reply:
@yosefe yes, [clang-tidy](https://clang.llvm.org/extra/clang-tidy/checks/readability/identifier-naming.html) can do that.